app/src/main/java/org/mifos/mobile/viewModels/RecentTransactionViewModel.kt

```diff
@@ -0,0 +1,91 @@
+package org.mifos.mobile.viewModels
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.observers.DisposableObserver
+import io.reactivex.schedulers.Schedulers
+import org.mifos.mobile.R
+import org.mifos.mobile.api.DataManager
+import org.mifos.mobile.models.Page
+import org.mifos.mobile.models.Transaction
+import org.mifos.mobile.utils.TransactionUiState
+import javax.inject.Inject
+
+class RecentTransactionViewModel @Inject constructor(
+    private val dataManager: DataManager?
```

Comment:
Repository? 😊 You might create one repository for all the Transaction related data.